ios/RNZoomUs.h

```diff
@@ -7,7 +7,15 @@
 
 #import <MobileRTC/MobileRTC.h>
 
-@interface RNZoomUs : NSObject <RCTBridgeModule, MobileRTCAuthDelegate, MobileRTCMeetingServiceDelegate>
```

Comment:
expand delegates to listen to in meeting events & add event emitter to send events to js